### ISSUE TITLE
Indistinguishable jw-settings-submenu panels

### ISSUE THREAD

**Issue by canvasplay** [jwplayer/jwplayer#3461]
### Expected Behavior

In terms of DOM, would be very useful to have an additional class to be able to differentiate a submenu from another.

````
<div class="jw-reset jw-settings-submenu jw-settings-submenu-{$name}" role="menu" aria-expanded="false">
  ...
</div>
````

From my point, this as easy as adding the additional when creating the panel.
[src/js/view/controls/components/settings/submenu.js#L12](https://github.com/jwplayer/jwplayer/blob/298d024196850e0b04af6f7094efc6ae8339884c/src/js/view/controls/components/settings/submenu.js#L12)

### Actual Behavior

There is no class, identifier or any reference in jw-settings-submenu panels to differentiate one from another. Which is the playbackRates submenu? Which is the captions submenu? They all look the same now:
````
<div class="jw-reset jw-settings-submenu" role="menu" aria-expanded="false">
  ...
</div>
````

### Steps to reproduce

Loading a very basic player, having quality selector is enough.
Just have a look at jw-settings-submenu elements inside the jw-settings-menu

### Environment

There are not specific environment conditions.

**Comment by DanFerrer** [comment]
Hi @canvasplay. All of the submenus all ready have unique identifiers in the DOM already (classes, name, and aria-label).

In the case of `playbackRates` for example, the line of code you referenced does append the string `playbackRates` to give it a class of `jw-submenu-playbackRates`.

When the element is fully created, it should look like this:

```<div class="jw-icon jw-icon-inline jw-button-color jw-reset jw-settings-playbackRates jw-submenu-playbackRates" role="menuitemradio" tabindex="0" aria-label="Playback Rates" style="" aria-checked="true" name="playbackRates">...```

If a video does have a settings menu, you should be able to find the submenus with on unique identifiers on with ```document.querySelector('.jw-settings-topbar').children```

**Comment by canvasplay** [comment]
Hi @DanFerrer. Yes, the buttons inside .jw-settings-topbar have an identifier class.
But, I am referring to the .jw-settings-submenu panels below the topbar.

**Comment by canvasplay** [comment]
I've created the following PR:
https://github.com/jwplayer/jwplayer/pull/3462

**Comment by DanFerrer** [comment]
Ah i see what you mean. Thanks for putting in the PR @canvasplay. We'll review it and keep you posted!

**Comment by robwalch** [comment]
@canvasplay merged #3462.
